src/classes.py

```diff
@@ -137,6 +160,7 @@ def __init__(self, no, length):
         self.length = length
         # important, it has to be sorted array
         self.cars = []
+        self.trucks = []
```

Comment:
Maybe we can put cars, trucks, and autonomous cars in the same array(f.e vehicles array)